src/test/java/com/game/Battleship/controllers/BoardControllerTests.java

```diff
@@ -25,7 +26,7 @@ public void boardEndpointContainsSizeAttribute(){
     }
 
     @Test
-    public void boardFieldSizeAttributeValueIs100(){
+    public void boardFieldSizeIsEqualWithBoardSizeToThePowerOf2(){
```

Comment:
Nice, now if your board size changes from 10 to 20, you wouldn't have to change the test description.